masterlist.yaml

```
@@ -3453,6 +3453,17 @@ plugins:
       - crc: 0xB9422B0D
         util: 'SSEEdit v4.0.3h (Hotfix 1)'
 
+  - name: 'MLP(SoulGems| \+ GIST Patch).*\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/25709/'
+        name: 'MultiLayer Parallax Soul Gems'
+    tag: [ Graphics ]
+    clean:
+      - crc: 0x31F662B9
+        util: 'SSEEdit v4.0.3h (Hotfix 1)'
+      - crc: 0x2FA1C27D
+        util: 'SSEEdit v4.0.3h (Hotfix 1)'
```

I'd only move the cleaning data into the regex if they're the same for both files.

```suggestion
  - name: 'MLP(SoulGems| \+ GIST Patch).*\.esp'
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/25709/'
        name: 'MultiLayer Parallax Soul Gems'
    tag: [ Graphics ]
  - name: 'MLPSoulGems.esp'
    clean:
      - crc: 0x31F662B9
        util: 'SSEEdit v4.0.3h (Hotfix 1)'
  - name: 'MLP + GIST Patch.esp'
    clean:
      - crc: 0x2FA1C27D
        util: 'SSEEdit v4.0.3h (Hotfix 1)'
```